countlog.h

```diff
@@ -35,7 +36,17 @@ static __thread pid_t _thread_id = 0;
 
 static pid_t get_thread_id() {
     if (_thread_id == 0) {
-        _thread_id = syscall(__NR_gettid);
+        #if TARGET_OS_MAC
```

Comment:
koala-libc does not work on mac. without interpose.h, the LD_PRELOAD way can not override all libc function under mac.